pkg/oidc/error.go

```diff
@@ -18,6 +18,7 @@ const (
 	InteractionRequired  errorType = "interaction_required"
 	LoginRequired        errorType = "login_required"
 	RequestNotSupported  errorType = "request_not_supported"
+	InvalidRefreshToken  errorType = "invalid_refresh_token"
```

Comment:
is this error in some RFC? i could only find invalid_grant for this case (https://www.rfc-editor.org/rfc/rfc6749#section-5.2):
> The provided authorization grant (e.g., authorization
               code, resource owner credentials) or refresh token is
               invalid, expired, revoked, does not match the redirection
               URI used in the authorization request, or was issued to
               another client.

Reply:
No, I made this up.  If that isn't allowed, I'll find another way to signal that the token isn't a refresh token.   I'll just return an extra bool insted.

Reply:
I mean I've found some other implementations (like slack) which return this error to the client, but i couldn't find some spec other than the one mentioned above. Therefore I just want to prevent that this error type is used publicly / as an error in a response. 
So i first thought about having a private error, but you obviously need a public one to be able to return it in the `GetRefreshTokenInfo` implementation.
What do you think about a simple error with a comment? Something like:
```
// ErrInvalidRefreshToken is used internally to state that the provided token is not a refresh token
// it must be returned in `GetRefreshTokenInfo`
var ErrInvalidRefreshToken = errors.New("refresh token is invalid")
```
I'm unsure where to declare this error, here or in storage.go, next to the `CanRefreshTokenInfo` interface

Reply:
I've moved the error to op/token_revocation.go.   Would you like it in op/storage.go instead?